### ISSUE TITLE
Support Python packages that aren't on PyPI

### ISSUE THREAD

**Issue by maxfisher-g** [ossf/package-analysis#755]
Packages such as https://github.com/ytisf/PyExfil have not been updated on PyPI and the current installation instructions recommend cloning from GitHub and running `pip setup.py install`. It would be nice if we could support analysis of Python packages with this kind of installation flow.

**Comment by dukecat0** [comment]
`pip setup.py install` is not a valid command. 
Usually `pip install git+https://github.com/ytisf/PyExfil.git` should work if the package needs to be installed from GitHub. This command will install it directly without needing to clone and run install commands manually.

**Comment by maxfisher-g** [comment]
Good point @dukecat0. I guess in general it would be nice to be able to override the install command in certain cases. This will be enabled by #301
